.gitlab-ci.yml

```diff
@@ -1,22 +1,41 @@
 stages:
+  - prepare
   - build
   - deploy-doc
 
 .env:
   before_script:
   - export DOCKER_IMAGE=`echo $CI_PROJECT_NAME | awk '{print tolower($0)}'`:$CI_COMMIT_SHORT_SHA
 
-build_lofar:
+build_base:
+  stage: prepare
+  extends: .env
+  script:
+  - docker build --tag dppp_base:20.04.0 -f ./docker/ubuntu_20_04_base .
+
+build-lofar:
   stage: build
+  dependencies: 
```

Comment:
Like in `build-test`, you want to make use of the base image, right?

So there probably should be something: `image: dppp_base:{YOUR_UPDATED_TAG}`